src/azure-cli/azure/cli/command_modules/appservice/_create_util.py

```diff
@@ -75,31 +72,31 @@ def zip_contents_from_dir(dirPath, lang):
     return zip_file_path
 
 
-def get_runtime_version_details(file_path, lang_name):
+def get_runtime_version_details(file_path, lang_name, is_linux=False, stack_helper=None):
```

Comment:
nit: should we make stack_helper a required parameter?

Reply:
yeah that's a good idea. It would just throw an attribute error if `stack_helper` was `None`
While I'm at it I'll also remove the function app stacks JSON file, like I did with the web app stacks file